examples/large_models/tp_llama/llama2_tokenizer.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Should we put the src files from llama2 into a subfolder to separate them from TS specific code?